CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## v1.1.0.0 - "Moore Features"
```

Comment:
I find it useful when changelogs include release dates, for things like trying to see how badly out of date the version you are currently using is, or for even finding a past version when you don’t know the exact version, but you know roughly when you obtained it.

Reply:
Good idea